src/main/views/privacy-policy.njk

```diff
@@ -1,122 +1,219 @@
 
       <p>
-        {{ t('As part of your claim we’ll ask if you want to receive notifications to let you know how '
-        + 'your application is proceeding. We’ll ask for your name and preferred contact details.') }}
+        {{ t('If another government department is involved in processing your claim or application, '
+        + 'we’ll share your data with them.') }}
+      </p>
+
+      <p>{{ t('If you contact us and ask for help with a service you are using, '
+        + 'your personal data may be shared with the Good Things Foundation. '
+        + 'This is a company who we have partnered with to offer face to face support.') }}
       </p>
 
       <p>
-        {{ t('This information is not shared with third parties and you can unsubscribe at any time '
-        + 'by phoning 0203 123 1234 or emailing mydata@hmcts.net.') }}
+        {{ t('We use Google Analytics to collect our website statistics. This anonymous website usage data is shared with Google.'
+        + 'Find out about this in our ' + internalLink('terms and conditions.', termsAndConditionsUri)) | safe }}
       </p>
 
       <h2 class="heading-medium">{{ t('Your rights') }}</h2>
 
-      <p> {{ t('You can ask:') }}</p>
+      <p>{{ t('You can ask:') }}</p>
 
       <ul class="list list-bullet">
         <li>{{ t('to see the personal data that we hold on you') }}</li>
-        <li>{{ t('to have the personal data corrected, removed or deleted') }}</li>
-        <li>{{ t('that access to this data is restricted (for example made temporarily unavailable)') }}</li>
-        <li>{{ t('that we hold your data for longer') }}</li>
+        <li>{{ t('to have the personal data corrected') }}</li>
+        <li>{{ t('to have the personal data removed or deleted (this will depend on the circumstances, for example if you decide not to continue your claim or application)') }}</li>
+        <li>{{ t('that access to the personal data is restricted (for example, you can ask to have your data stored for longer and not automatically deleted)') }}</li>
       </ul>
 
-      <h2 class="heading-medium">{{ t('Disclaimer') }}</h2>
+      <p>
+        {{ t('If you want to see the personal data that we hold on you, '
+        + internalLink('you can make a subject access request.', dataRequestUri)
+        +' This request goes to the MoJ as data controller.') | safe }}
+      </p>
+
+      <h2 class="heading-medium">{{ t('How to complain') }}</h2>
+
+      <p>
+        {{ t('See our '+ internalLink('complaints procedure,', complaintsProcedureUri)
+        + ' if you want to complain about how we’ve handled your personal data.') | safe }}
+      </p>
 
       <p>
-        {{ t('While we make every effort to keep GOV.UK up to date, we don’t provide any guarantees, '
-        + 'conditions or warranties that the information will be:') }}
+        {{ t('You can also complain to the ' + internalLink('Information Commissioner’s Office' , informationCommissionerOfficeUri)
```

Comment:
why is this an internalLink?
normally all links off our site are externalLinks?

Reply:
I completely agree. I was following the prototype to the letter.

Reply:
following prototype to the letter is never a good idea, please confirm with Ronan/ Julian / Tarik
Although you will probably struggle to get hold of Tarik

Reply:
We need to bring this up in our retro then. If i cant trust the work given to me to copy, then it taints the whole concept of prototype.